ProteGO/Common/Bluetooth/Scanner/Device.swift

```diff
@@ -0,0 +1,343 @@
+import Foundation
+import CoreBluetooth
+
+/// This class provides information about the device which was discovered by
+/// a central manager. Only device ID is a stable value.
+class Device {
```

Comment:
Może `ProtegoPeripheral`? Czy klasa jest używania nie tylko dla tych?

Reply:
Zgadza się, klasa jest specyficzna dla ProteGo

Reply:
`Device` jako taki nie precyzuje czy występuje tu w kontekście `Peripheral` czy `Central`. Nie jest też jasnym od razu czy jest to urządzenie które zostało już przefiltrowane pod kątem bycia użytecznym dla aplikacji czy nie. Co sądzisz?